src/mon/Session.h

```diff
@@ -159,7 +159,6 @@ struct MonSessionMap {
     if (s->con_features) {
       feature_map.add(s->con_type, s->con_features);
     }
-    s->get();  // caller gets a ref
```

Comment:
I'm pretty sure the caller of `new_session` was leaking this ref in some spots. This was due to `con->set_priv` not consuming the ref in some spots. I've just removed the `->get()` here. The caller in responsible for incrementing the ref if necessary.